lib/locales.js

```diff
@@ -89,6 +89,11 @@ const locales = [
     code: "pt_BR",
     icon: "pt_BR",
   },
+  {
+    name: "বাংলা",
+    code: "bn",
+    icon: "bangladesh",
```

Comment:
You should change this to `bd` as the icon's filename is `bd.svg`.
```suggestion
    icon: "bd",
```

Reply:
ok I got you